main.go

```diff
@@ -74,10 +69,12 @@ Sample usage:
 func main() {
 	flag.Usage = usage
 	flag.Parse()
+
 	if webapi {
```

Comment:
Not related to this PR, but what about adding a comment:

// Note: the web interface does not use any commandline flags.

Reply:
Done @https://github.com/cloudspannerecosystem/harbourbridge/pull/121/commits/96fafc8df69686a5aeac12ee5bec2c5656666d1e.